src/DIRAC/Resources/Computing/BatchSystems/Condor.py

```diff
@@ -85,25 +85,17 @@ def submitJob(self, **kwargs):
         if not nJobs:
             nJobs = 1
         numberOfProcessors = kwargs.get("NumberOfProcessors")
-        wholeNode = kwargs.get("WholeNode")
```

Comment:
How about:
```
        if kwargs.get("WholeNode"):
            return an error
```

Reply:
To make a transition in case someone would still be using the option?
Indeed, we could add an error/warning to inform the users in `v7r3` and then delete it in `v8`.
Is that what you are proposing here?

Reply:
https://github.com/DIRACGrid/DIRAC/pull/6604/files#diff-d322ec15aced879e80a750fa4f7124ba0a740ea24651faf46f64975e6fbe2a75R92-R97